client/components/plans/plan-price/style.scss

```diff
@@ -89,3 +89,25 @@
 		@include plans-in-three-columns();
 	}
 }
+
+.jetpack-price {
+	margin-top: 20px;
+	padding: 0 6px;
+	display: inline-block;
+	position: relative;
+}
+
+.jetpack-price:first-child {
+	opacity: 0.4;
+}
+
+.jetpack-price:first-child::before {
+	content: '';
+	border-bottom: 2px solid red;
+	width: 100%;
+	position: absolute;
+	right: 0;
+	top: 50%;
```

Comment:
`right` and `top` should be indented, according to the docs (https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md?term=)